src/components/App/App.js

```diff
@@ -48,9 +51,19 @@ class App extends Component {
         {/* {this.state.error && <h2>{this.state.error}</h2>} */}
         <Header />
         <main>
-          <Hero randomMovie={this.state.movieForHero} />
+          <Switch>
+            <Route exact path="/">
 
-          <MovieContainer movieData={this.state.movies} />
+              <Hero randomMovie={this.state.movieForHero} />
+              <MovieContainer movieData={this.state.movies} />
+            </Route>
+          
+           <Route path="/:id" 
```

Comment:
Game Changer right hurr! The match.params is by far one of my favorite features of React Router 👍 🔥